lib/tools/system-calls.js

```diff
@@ -236,6 +230,7 @@ systemCallMethods.restartAdb = async function restartAdb () {
   log.debug('Restarting adb');
   try {
     await this.killServer();
+    await this.adbExec(['start-server']);
```

Comment:
ah, we did not start the server even if we named 'restart'

Reply:
I assume we relied on the following command. E.g. sending it should also make adb to start automatically